test/rule_test_support.dart

```diff
@@ -214,32 +215,6 @@ abstract class LintRuleTest extends PubPackageResolutionTest {
       fail(buffer.toString());
     }
   }
```

Comment:
Hrmm.  Looks like I didn't rebase.  Luckily the change that removes `assertLint ` has landed.  Sorry for the noise!

Reply:
np

Reply:
Thanks!